### PR TITLE
[IMP] document_quick_access_folder_auto_classification: Add Python-OpenCV as backup check

### PR DESCRIPTION
**Pull request by etobella**
#530

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 odoo_test_helper
+opencv-python
```
Comment (nilshamerlinck) [inline test-requirements.txt]: Hello sir, fyi `opencv-python-headless` is [more suitable for servers](https://github.com/opencv/opencv-python#installation-and-usage) (doesn't require `libgl1` deb package)

image

Comment (etobella) [reply test-requirements.txt]: I see, I didn't know, thanks for that :smile:

**Comment by bosd** [comment]
Can you re-create the runboat?

**Comment by etobella** [comment]
Sure! Rebase applied :smile:

**Review by kluna1998 — APPROVED** [review]
LGTM!

**Review by bosd — APPROVED** [review]
Okay, Let's go! 🚀

**Pull request merged**